mod.hjson

```diff
@@ -17,4 +17,4 @@ description: "A Mindustry Kotlin mod template."
 version: 1.0
 
 #the minimum game build required to run this mod
-minGameVersion: 129
+minGameVersion: 136
```

Comment:
Uhh don't you mean a minGameVersion of 146?

Reply:
Yeah